### PR TITLE
Add coverage to CI

### PR DESCRIPTION
**Pull request by emcastillo**

### CODE REVIEW

**Comment by emcastillo** [comment]
creating image
https://ci.preferred.jp/pytorch-pfn-extras.prep-linux/75425/

File: .flexci/linux/script.sh [hunk 1]
```diff
@@ -47,6 +47,8 @@ main() {
           /src/.flexci/linux/unittest.sh "${TARGET}"
       gsutil -m -q cp /tmp/output/pysen.txt gs://${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/pysen.txt
       echo "pysen output: https://storage.cloud.google.com/${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/pysen.txt"
+      gsutil -m -q cp -r /tmp/output/htmlcov gs://${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/htmlcov
+      echo "pysen output: https://storage.cloud.google.com/${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/htmlcov"
```
Comment (emcastillo) [inline .flexci/linux/script.sh:51]:
```suggestion
      echo "pytest-cov output: https://storage.cloud.google.com/${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/htmlcov"
```

File: .flexci/linux/script.sh [hunk 2]
```diff
@@ -47,6 +47,8 @@ main() {
           /src/.flexci/linux/unittest.sh "${TARGET}"
       gsutil -m -q cp /tmp/output/pysen.txt gs://${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/pysen.txt
       echo "pysen output: https://storage.cloud.google.com/${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/pysen.txt"
+      gsutil -m -q cp -r /tmp/output/htmlcov gs://${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pysen/${CI_JOB_ID}/htmlcov
+      echo "pytest-cov output: https://storage.cloud.google.com/${PPE_FLEXCI_GCS_BUCKET}/pytorch-pfn-extras/pytest-cov/${CI_JOB_ID}/htmlcov/index.html"
```
Comment (emcastillo) [inline .flexci/linux/script.sh:51]:
```suggestion
      echo "pytest-cov output: https://storage.googleapis.com/chainer-artifacts-pfn-public-ci/pytorch-pfn-extras/pytest-cov/${CI_JOB_ID}/htmlcov/index.html"
```

**Comment by kmaehashi** [comment]
Thanks!

**Pull request merged**